app/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     defaultConfig {
-        applicationId "io.github.djunicode.salespersontracker"
-        minSdkVersion 19
-        targetSdkVersion 28
+        applicationId "com.example.salespersontracker"
```

Comment:
The package name should be `io.github.djunicode.salespersontracker`. This needs to be changed everywhere.
Also please send a PR to the `dev` branch and not the `master` branch.